app/src/main/java/org/oppia/app/hintsandsolution/HintsAndSolutionAdapter.kt

```diff
@@ -132,7 +135,9 @@ class HintsAndSolutionAdapter(
               position
             }
           expandedHintListIndexListener.onExpandListIconClicked(currentExpandedHintListIndex)
-          if (previousIndex != null && currentExpandedHintListIndex != null && previousIndex == currentExpandedHintListIndex) {
+          if (previousIndex != null && currentExpandedHintListIndex != null &&
+            previousIndex == currentExpandedHintListIndex
```

Comment:
Nit here and below:
```
          if (previousIndex != null 
            && currentExpandedHintListIndex != null 
            && previousIndex == currentExpandedHintListIndex
```

Reply:
ktlint says line cannot start with `&&`

Reply:
Then it should be:
```
          if (previousIndex != null &&
            currentExpandedHintListIndex != null &&
            previousIndex == currentExpandedHintListIndex
```

Reply:
You can put the && at the end of the line instead then
i.e.  
```
if (previousIndex != null &&
    currentExpandedHintListIndex != null && 
    previousIndex == currentExpandedHintListIndex
```

Reply:
Sounds good

Reply:
Done